src/components/Onboarding/utils/progress-utils.ts

```diff
@@ -1,19 +1,19 @@
 import { getNetworkType } from '@utils/web3-utils'
 import { dataURLtoFile, textToFile } from '@utils/kit-utils'
 
-const getStorageKey = (account, chainId) =>
+const getStorageKey = (account: string, chainId: number) =>
   `onboarding:${getNetworkType(chainId)}:${account}`
 
-export const getItem = (account, chainId) => {
+export const getItem = (account: string, chainId: number) => {
   const item = window.localStorage.getItem(getStorageKey(account, chainId))
   return item ? JSON.parse(item) : null
 }
 
-export const removeItem = account => {
-  window.localStorage.removeItem(getStorageKey(account))
+export const removeItem = (account: string, chainId: number) => {
```

Comment:
The problem happens here. Its called removeItem passing chainID but the method just accepted one argument,so chainID was undefined.

Migratiing to TS it show on compile time.